### DISCUSSION TITLE
Account Recovery

### DISCUSSION THREAD

**Discussion by realbill2017** [community/community#110292] [New to GitHub]
Hi! I hope anyone does not encounter this problem. I opened github and my account was logged out. I cant login to my account so i used "Forgot Password" option. When I reset my password and tried to login, a new account has been created. Now i lost all of my projects. Is there a way to recover my old account? I can still see it when i click "See all accounts" option on my newly (accidentally) created account.

**Comment by thekarthikr** [comment]
# GitHub Account Recovery Guide

If you've accidentally lost access to your GitHub account and created a new one, don't worry. Follow these steps to recover your old account:

1. **Contact GitHub Support**: 
   - Go to the [GitHub Support page](https://support.github.com/contact) and explain your situation. Provide as much information as possible about your old account.
   
2. **Provide Proof of Ownership**:
   - GitHub may ask for proof that you own the old account. This could include details like the account's email address, recent activity, or linked social media accounts.

3. **Use Account Recovery Options**:
   - GitHub might have recovery options available. Follow any instructions provided to verify your identity and regain access to your old account.

4. **Merge Accounts (if applicable)**:
   - If GitHub confirms both accounts belong to you, they may be able to merge them, restoring access to your projects and data.

5. **Prevent Future Issues**:
   - Once you regain access, update your recovery options, enable two-factor authentication, and regularly backup your important data to prevent future issues.

By following these steps and working with GitHub support, you should be able to recover your old account and access your projects again.

**Reply by realbill2017** [reply]
wow thanks @thekarthikr

**Comment by Yerock09** [comment]
Thank you for your help!
